test/integration/SubscriptionManager.test.js

```diff
@@ -102,20 +98,24 @@ describe('SubscriptionManager', () => {
         await waitForCondition(() => broker1.getStreams().length === 2)
         await waitForCondition(() => broker2.getStreams().length === 2)
 
-        await wait(2000)
-
         expect(broker1.getStreams()).toEqual([freshStreamId1 + '::0', freshStreamId2 + '::0'].sort())
         expect(broker2.getStreams()).toEqual([freshStreamId1 + '::0', freshStreamId2 + '::0'].sort())
 
         await client1.subscribe({
             stream: freshStreamId1
         }, () => {})
 
-        await wait(2000)
+        await waitForCondition(() => broker1.getStreams().length === 2)
```

Comment:
Hmm.. before it is already checked that
```js
expect(broker1.getStreams()).toEqual([freshStreamId1 + '::0', freshStreamId2 + '::0'].sort())
expect(broker2.getStreams()).toEqual([freshStreamId1 + '::0', freshStreamId2 + '::0'].sort())
```
which implies length == 2 for both broker1 and broker2.